_pydevd_bundle/pydevd_comm.py

```diff
@@ -994,9 +994,12 @@ def do_it(self, dbg):
             val_dict = pydevd_vars.resolve_compound_variable(self.thread_id, self.frame_id, self.scope, self.attributes)
             if val_dict is None:
                 val_dict = {}
-                
-            keys = dict_keys(val_dict)
-            keys.sort(key=compare_object_attrs_key)
+
+            # assume properly ordered if resolver returns 'OrderedDict'
+            # check type as string to support OrderedDict backport for older Python
+            if not str(type(val_dict)).endswith("OrderedDict"):
```

Comment:
Instead of str(type()), I think you could use: 

`a.__class__.__name__`

Also, with this patch, `keys` will not be defined if it actually is an `OrderedDict`.